src/inc/tetris_board.h

```diff
@@ -31,6 +32,12 @@ class TetrisBoard : public GameComponent
 
   void add_piece( TetrisPiece& piece );
 
+  int score();
+
+  int get_score();
+
+  bool new_score();
```

Comment:
This method is just a temporary way to only read the score if it has been updated by the board. I would like to make state machine for the board that will handle scoring and other board events.